dotcom-rendering/src/footballMatches.test.ts

```diff
@@ -170,4 +170,29 @@ describe('footballMatches', () => {
 
 		expect(result.kind).toBe('UnexpectedLiveMatch');
 	});
+	it('should return a clean team name', () => {
+		const matchesListWithTeamName = (teamName: string): FEResult => {
+			return {
+				...matchResult,
+				homeTeam: {
+					...matchResult.homeTeam,
+					name: teamName,
+				},
+			};
+		};
+
+		const uncleanToCleanNames: Record<string, string> = {
+			Ladies: '',
+			Holland: 'The Netherlands',
+			'Union Saint Gilloise': 'Union Saint-Gilloise',
+		};
+
+		Object.entries(uncleanToCleanNames).map(([uncleanName, cleanName]) => {
```

Comment:
```suggestion
		Object.entries(uncleanToCleanNames).forEach(([uncleanName, cleanName]) => {
```
seems more standard here

Reply:
Yea I just pushed the change for this :)